READMEs/search-and-filter-assets.md

```diff
@@ -0,0 +1,87 @@
+<!--
+Copyright 2022 Ocean Protocol Foundation
+SPDX-License-Identifier: Apache-2.0
+-->
+
+
+# Quickstart: Search Assets Flow
+
+This quickstart describes how assets can be found by their `tags` from Aquarius.
+
+
+Here are the steps:
+
+1.  Setup
+2.  Alice creates few assets for testing
+3.  Alice searches & filters assets by their `tags`
+
+Let's go through each step.
+
+## 1. Setup
+
+From [data-nfts-and-datatokens-flow](data-nfts-and-datatokens-flow.md), do:
+- [x] Setup : Prerequisites
+- [x] Setup : Download barge and run services
+- [x] Setup : Install the library
+- [x] Setup : Set envvars
+- [x] Setup : Setup in Python
+
+
+## 2. Alice publishes datasets
+
+Now, you're Alice. Using [publish-flow](publish-flow.md) model, do:
+
+```python
+#data info
+name = "Branin dataset"
+url = "https://raw.githubusercontent.com/trentmc/branin/main/branin.arff"
+
+# Created a list of tags for the following assets
+tags = [
+    ["test", "ganache", "best asset"],
+    ["test", "ocean"],
+    ["AI", "dataset", "testing"],
+]
+# Publish few assets for testing
+for i in range(len(tags)):
+
+    (data_NFT, datatoken, asset) = ocean.assets.create_url_asset(name, url, alice_wallet)
+    print(f"Just published asset, with did={asset.did}")
+    
+    # Update the metadata introducing `tags`
+    asset.metadata.update({
+        "tags": tags[i]
+    })
+    updated_asset = ocean.assets.update(asset=asset, publisher_wallet=alice_wallet, provider_uri=config["PROVIDER_URL"])
+    print(f"Just updated the metadata of the asset with did={updated_asset.did}. New metadata is: {updated_asset.metadata}")
```

Comment:
Tweak: remove the part `New metadata is: {updated_asset.metadata}`. Why: it's unneeded, and would be a large printout which hinders usability.